inference-engine/src/transformations/src/transformations/convert_divide.cpp

```diff
@@ -18,9 +18,9 @@ void ngraph::pass::ConvertDivide::convert_divide() {
     ngraph::graph_rewrite_callback callback = [](pattern::Matcher& m) {
         auto div = std::dynamic_pointer_cast<ngraph::opset1::Divide> (m.get_match_root());
         // We can not apply this transformation in case with integer input data type
-        if (!div || div->input(0).get_element_type().is_integral()) {
+//        if (!div || div->input(0).get_element_type().is_integral()) {
```

Comment:
I know that this is draft but if you even disable this decomposition via callback you will probably have performance degradation as we do not fuse Divide/Subtract operations into Convolution or FC directly. And in this case I would suggest to have a transformations that finally fuses sequence of ops to Divide or Subtract.

Reply:
This is temporary change that needed only to test Divide operation implementation inside the plug-in. I will revert it before merge.